charms/kfp-persistence/tests/unit/test_operator.py

```diff
@@ -21,9 +21,15 @@ def harness():
     return harness
 
 
-def test_not_leader(
-    harness,  # noqa F811
-):
+@pytest.fixture()
```

Comment:
@DnPlas, apparently I can't just do that due to [this](https://github.com/canonical/kfp-operators/actions/runs/6182549418/job/16782510545?pr=324#step:4:26), since `mocker` is function-scoped. I can either leave this as is (like we do in the rest of the KFP charms) or use the patch decorator for each one of the tests directly, but then it's function-scoped again. I think we should just leave it as is.

Reply:
Oh, I see. I think then you could use the `module` scope, but it's fine, it's a minor thing.